src/widgets.js

```diff
@@ -197,6 +197,38 @@ define([
         return widget;
     };
 
+    var multilineText = function (mug, options) {
+        var widget = normal(mug, options);
+
+        widget.input = $("<textarea></textarea>")
```

Comment:
Eh. I think it would make more sense to have `normal` not create an input at all, and if you want an `input` you would inherit from `text`, `textarea` from `multilineText`, etc.

Going to pass on that sort of refactor for this PR though